docs/contributing.md

```diff
@@ -0,0 +1,115 @@
+# Contibuting
+
+All contributions to *databases* are welcomed!
+
+## Issues
+
+To make it as simple as possible for us to help you, please include the following:
+
+* OS 
+* python version
+* databases version
+* database backend (mysql, sqlite or postgresql)
+* database driver (aiopg, aiomysql etc.)
+
+Please try to always include the above unless you're unable to install *databases* or **know** it's not relevant
+to your question or feature request.
+
+## Pull Requests
+
+It should be quite straight forward to get started and create a Pull Request.
+
+!!! note
+    Unless your change is trivial (typo, docs tweak etc.), please create an issue to discuss the change before
+    creating a pull request.
+
+To make contributing as easy and fast as possible, you'll want to run tests and linting locally. 
+
+You'll need to have **python >= 3.6 (recommended 3.7+)** and **git** installed.
+
+## Getting started
+
+1. Clone your fork and cd into the repo directory
+```bash
+git clone git@github.com:<your username>/databases.git
+cd databases
+```
+
+2. Create and activate virtual env
```

Comment:
Can we use `./scripts/install` instead of this?

Reply:
We can but by default, those scripts will not run on windows. 
You need two steps for it:
* #454 
*  add some Linux like shell into windows path, i.e. git `sh.exe` file

That's why I kept the raw commands, but we can change this

Reply:
Yeah, I guessed so. So can we remove the `python 3.6` part from the virtual env? We can rely on the system default?

Reply:
Done